tests/testthat/test-PreparePlottingData.R

```diff
@@ -197,200 +203,6 @@ test_that("all columns besides 'term' and 'label' are 0 for the normalization co
 
 # FHS ---------------------------------------------------------------------
 
-test_that("labels are unique", {
```

Comment:
Why did we drop this chunk of tests?

Reply:
I agree with you @santiagohermo!

Reply:
Thanks @nateschor! I'll resolve @jmshapir, but feel free to reopen if you have other thoughts.